docs/guide/dev/integration/react/v1/README.md

```diff
@@ -20,7 +20,7 @@ For licensing, see LICENSE.md.
 	This feature is provided through the <a href="https://www.npmjs.com/package/ckeditor4-react"><code>ckeditor4-react</code> npm package</a>.
 </info-box>
 
-CKEditor 4 offers a native React integration through the CKEditor 4 React component. It provides deep integration of CKEditor 4 and React that lets you use the native features of the WYSIWYG editor inside a React component. The CKEditor 4 React component is compatible with React versions higher than v16.0.x but lower than v17.x. Although it seems to be working with React v17.x, we do not provide any support in that regard.
+CKEditor 4 offers a native React integration through the CKEditor 4 React component. It provides deep integration of CKEditor 4 and React that lets you use the native features of the WYSIWYG editor inside a React component. The CKEditor 4 React component is compatible with React versions higher than v16.0.x but lower than v16.8. For React 16.8 or higher use {@link guide/dev/integration/react/v2/README CKEditor 4 React Integration - v2}
```

Comment:
To be precise, `ckeditor4-react` v1 is also compatible with React v17. Therefore, I would slightly change the wording here.

```suggestion
CKEditor 4 offers a native React integration through the CKEditor 4 React component. It provides a deep integration of CKEditor 4 and React that lets you use the native features of the WYSIWYG editor inside a React component. The CKEditor 4 React component is compatible with React v16.x and higher. However, for React 16.8 and higher we strongly advise to use {@link guide/dev/integration/react/v2/README CKEditor 4 React Integration - v2}.
```

Reply:
Good point 👍  corrected.